examples/game/lib/constraint.dart

```diff
@@ -41,3 +42,29 @@ class ConstraintRotationToMovement {
       node.rotation = _dampenRotation(node.rotation, target, dampening);
   }
 }
+
+class ConstraintRotationToNode {
+  ConstraintRotationToNode(this.targetNode, [this.baseRotation, this.dampening]);
+
+  final Node targetNode;
+  final double baseRotation;
+  final double dampening;
+
+  void constrain(Node node, double dt) {
+    Offset offset;
+
+    if (targetNode.parent == node.parent) {
+      offset = targetNode.position - node.position;
+    } else {
+      offset = node.convertPointToBoxSpace(Point.origin)
+        - targetNode.convertPointToBoxSpace(Point.origin);
+    }
+
+    double target = degrees(GameMath.atan2(offset.dy, offset.dx)) + baseRotation;
+
+    if (dampening == null)
```

Comment:
You could write this as a ternary if you want:

node.rotation = (dampening == null) ? target : _dapenRotation(node.rotation, target, dampening);

Alternatively you could just make dampenRotation handle null dampening.  That's probagbly what I would do.